multicluster/apis/multicluster/v1alpha1/resourceimport_types.go

```diff
@@ -65,9 +65,38 @@ type ResourceImportSpec struct {
 	Raw *RawResourceImport `json:"raw,omitempty"`
 }
 
+type ResourceImportConditionType string
+
+const (
+	ResourceImportSucceeded ResourceImportConditionType = "Succeeded"
+)
+
+// ResourceImportCondition indicates the condition of the ResourceImport in a cluster
+type ResourceImportCondition struct {
```

Comment:
But do you think later we can have overall ResourceImportCondition, in addition to the per Cluster Conditions? I do not have an opinion what naming is better; just like to check what you think.

Reply:
that was my original intention that's why named it with Cluster in it originally. Maybe if we decide to introduce overall status later, something like `ResourceImportOverallStatus` or `ResourceImportClusterSetStatus`, if we want to keep this (`ResourceImportCondition`) naming today to be consistent with the `ResourceExportCondition`